base/sparse/cholmod.jl

```diff
@@ -1464,7 +1464,7 @@ Ac_ldiv_B(L::FactorComponent, B) = ctranspose(L)\B
 
 (\){T}(L::Factor{T}, B::Dense{T}) = solve(CHOLMOD_A, L, B)
 (\)(L::Factor{Float64}, B::VecOrMat{Complex{Float64}}) = L\real(B) + L\imag(B)
-(\)(L::Factor, b::Vector) = Vector(L\convert(Dense{eltype(L)}, b))
+(\)(L::Factor, b::StridedVector) = Vector(L\convert(Dense{eltype(L)}, b))
 (\)(L::Factor, B::Matrix) = Matrix(L\convert(Dense{eltype(L)}, B))
```

Comment:
This one should also be changed to `StridedMatrix` although it's not needed for `eigs`.